eva-lib/pom.xml

```diff
@@ -15,24 +15,49 @@
         <dependency>
             <groupId>org.opencb.opencga</groupId>
             <artifactId>opencga-storage-mongodb</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.mongodb</groupId>
+                    <artifactId>mongo-java-driver</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>uk.ac.ebi.eva</groupId>
             <artifactId>variation-commons</artifactId>
             <version>0.2-SNAPSHOT</version>
```

Comment:
Given that `variation-commons` is an internal dependency, it would be best to upgrade the Spring Boot starters to the versions that we want to use, instead of excluding them.

The OpenCGA exclusion is fine because we will remove that dependency at some point.